### PR TITLE
Fixes for latest pkg/http_parser

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## 0.6.8
+
+* Require the latest version of `package:http_parser`.
```
Comment (nex3) [inline CHANGELOG.md]: I don't like requiring patch versions, even if they fix relevant bugs. There's no reason the version solver should ever select 3.1.0 anymore anyway.

Also, if we did do this, we'd release it as 0.6.7+3, not 0.6.8.

Comment (kevmoo) [inline CHANGELOG.md]: I was hoping to help folks who are just running `pub get` – they may still get the old thing.

Also, I was planning to push a few more commits before I wanted to release – agree w/ the eventual version, though.

Comment (nex3) [inline CHANGELOG.md]: If users just run "pub get", they'll also get the older `shelf` version, so I don't think there's much help there.

**Comment by kevmoo** [comment]
@nex3 PTAL

**Comment by kevmoo** [comment]
I'm OOO. Please merge :-)

**Pull request merged**